libs/build_api_v2.py

```diff
@@ -35,17 +39,52 @@
 USA_VACCINATION_START_DATE = datetime(2020, 12, 14)
 
 
-def _build_actuals(actual_data: dict) -> Actuals:
+_logger = structlog.get_logger()
+
+
+def _filter_demographic_field_and_remove_prefix(demographic_field: str, data: Dict):
+    return {
+        key.replace(demographic_field + ":", ""): value
+        for key, value in data.items()
+        if key.startswith(demographic_field)
+    }
+
+
+def _build_demographic_data_for_field(
+    field_bucket: Dict[str, int]
+) -> Optional[DemographicDistributions]:
+
+    data = {
+        field: _filter_demographic_field_and_remove_prefix(field, field_bucket) or None
+        for field in can_scraper_helpers.DEMOGRAPHIC_FIELDS
+    }
+
+    # If there is no demographic data, do not create a DemographicDistributions
```

Comment:
What about the default, `all`?

Reply:
that's not included in the demographic specific data - essentially the normal `actuals` data is all of the all data